managed/services/grafana/auth_server.go

```diff
@@ -214,6 +217,9 @@ func (s *AuthServer) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 			l.Warnf("%s", err)
 		}
 	}
+
+	// Adds a header indicating full access to all metrics when used with vmgateway.
+	rw.Header().Set("X-Percona-Token", "Bearer "+jwtFullAccess)
```

Comment:
I assume that we need to have a middleware/proxy to remove this header for the end user and add it only for a vmgateway. In that case, adding vmgateway makes no sense in terms of security.